templates/zerver/faq.html

```diff
@@ -102,6 +102,32 @@ <h1>Frequently asked questions</h1>
                     open source project or similar organization.
                 </p>
             </div>
+            <div class="faq">
+                <div class="question">
+                    What is the difference between automatic and manual billing?
+                </div>
+                <p class="answer">
+                    With automatic billing, you only pay for the current number of
+                    users in your organization. With manual billing, you choose a
+                    preset user limit. If the limit is reached, no more users can
+                    join until additional licenses are paid for. In general, we
```

Comment:
> additional licenses are **paid** for

They do have to pay for the additional users. But the invoice/charge is only made at the end of billing period.

Reply:
Hmm, maybe "requested" rather than "paid for" is more accurate, then? Or "ordered"?

Reply:
When they increase the licenses in /billing, the additional licenses are added instantly to the account but we make the charge only at the end of the billing period. Ahh.

How about

"no more users can join until additional licenses are added"
 
I think issue with "ordered" is it makes it sound like the process is not instant.